js/book_list.js

```diff
@@ -68,6 +79,32 @@ class Storage {
   }
 }
 
+class ShowViews {
+  static clear() {
+    mainContent.textContent = '';
+  }
+
```

Comment:
* The current date is not displayed below the navigation bar as stated in the project requirement, Kindly add the current date using the [Luxon Library](https://moment.github.io/luxon/#/install).
*  [See here on how to get current Date and Time](https://moment.github.io/luxon/docs/manual/tour.html#get-the-current-date-and-time)